frontend/src/components/FilterForm/FilterForm.tsx

```diff
@@ -20,8 +21,12 @@ function FilterForm({ toggleFilter }: FilterFormProps) {
     setFilters(newFilters);
   };
 
-  const handleChangeRemoteWork = (event: React.ChangeEvent<HTMLInputElement>) => {
-    const newFilters = { ...filters, isRemote: event.target.checked };
+  // const handleChangeRemoteWork = (event: React.ChangeEvent<HTMLInputElement>) => {
+  //   const newFilters = { ...filters, isRemote: event.target.checked };
+  //   setFilters(newFilters);
+  // };
```

Comment:
Hey! Please delete the commented code if we won't use it :)

Reply:
ok..